packages/flutter/lib/src/material/snack_bar_theme.dart

```diff
@@ -115,6 +116,12 @@ class SnackBarThemeData with Diagnosticable {
   /// [SnackBarBehavior.floating].
   final double? width;
 
+  /// Default value for [SnackBarIcon.icon].
+  ///
+  /// If null, and an [icon] property is included in the snack bar constructor,
+  /// a default 'Close' icon will display.
+  final Icon? icon;
```

Comment:
Perhaps this should be just a `Widget?` which would be a little more flexible (i.e. they could wrap an `Icon` with some padding or some such).